resources/js/features/authentication/pages/Login.page.tsx

```diff
@@ -0,0 +1,101 @@
+import { SubmitHandler, createForm, zodForm } from "@modular-forms/solid";
+import { Navigate } from "@solidjs/router";
+import { createMutation, useQueryClient } from "@tanstack/solid-query";
+import { Button } from "components/ui";
+import { Page, QueryBarrier } from "components/utils";
+import { User } from "data-access/memo-api";
+import { Component } from "solid-js";
+import { z } from "zod";
+
+namespace LoginForm {
+  export const schema = z.object({
+    email: z
+      .string()
+      .min(1, "Pole jest wymagane")
+      .email("Wprowadź poprawny email"),
+    password: z.string().min(1, "Pole wymagane"),
+  });
+
+  export const initialValues: Input = {
+    email: "",
+    password: "",
+  };
+
+  export type Input = z.input<typeof schema>;
+  export type Output = z.output<typeof schema>;
+}
+
+const Content: Component = () => {
+  const [, { Form, Field }] = createForm<LoginForm.Input>({
+    initialValues: LoginForm.initialValues,
+    validate: zodForm(LoginForm.schema),
+  });
+  const queryClient = useQueryClient();
+  const mutation = createMutation({
+    mutationFn: User.login,
+    onSuccess() {
+      queryClient.invalidateQueries(User.keys.status());
+    },
+  });
+
+  const handleSubmit: SubmitHandler<LoginForm.Output> = (values) => {
+    mutation.mutate({
+      ...values,
+    });
+  };
+
+  return (
+    <Page title="Logowanie">
+      <div
+        class="min-h-screen h-screen max-h-screen flex justify-center items-center"
+        style={{ "background-color": "#f7f3e7" }}
```

Comment:
Te różne kolorki to są starannie dobrane i niezmienne tak że mają docelowo zostać inline rozsiane po kodzie? Czy raczej chcemy je gdzieś potem zebrać w jednym miejscu, w stylach albo w stałych?

Reply:
na ten moment są hardcodowane, chciałbym utworzyć globalną konfigurację palety kolorów, ale szczerze nie mam na to głowy :D nie mam pomysłu jakie faktycznie kolory by pasowały, a trochę ich by było. Można to obgadać na jakimś spotkanku